### PR TITLE
tg-beta-issue-15

### PR DESCRIPTION
**Pull request by nalwa**
This PT has fix for the beta-issue-15 related to promoted email.

@gautamv16  and @akuks  please review and merge PR.

### CODE REVIEW

File: resources/views/emails/promotedDelegatesToDirectMail.blade.php [hunk 1]
```diff
@@ -22,7 +22,7 @@
...
+               <td>They have entirely removed their support of all camps in this topic, so you have been promoted to a direct supporter in their place. Direct supporters are expected to participate in the maintenance of camps, including receiving and where necessary, responding to emails regarding the maintenance of directly supported camps. If you are not able to do this, you can delegate your support to any other supporter in the <a href="<?= $data['camp_link']; ?>" target="_blank"><?= $data['camp']->camp_name; ?></a> camp. Or you can entirely remove your support of all camps <a href="<?= $data['support_link']; ?>" target="_blank">here</a>.</a></td>
```
Comment (akuks) [inline resources/views/emails/promotedDelegatesToDirectMail.blade.php:25]: Please move it to message file

**Review by akuks — CHANGES_REQUESTED** [review]
@nalwa Please do the changes.

File: resources/views/emails/promotedDelegatesToDirectMail.blade.php (hunk 1 above)
Comment (akuks) [reply resources/views/emails/promotedDelegatesToDirectMail.blade.php:25]: @nalwa Have you got a chance to look at this?

Comment (nalwa) [reply resources/views/emails/promotedDelegatesToDirectMail.blade.php:25]: @akuks  we are not using any email content from message anywhere. why this needs to be moved to message file? i am not sure about it.

**Review by akuks — APPROVED** [review]

**Pull request merged**